ops/travis/unit-testing/python-linting

```diff
@@ -15,22 +15,17 @@ before_install() {
 install() {
     if bash ops/travis/helpers/test_for_changes "${FOLDER_CHECKS[@]}";
     then
-        pip install --upgrade pip wheel setuptools && pip3 --version
-        pip3 install packages/s3wrapper[test]
-        pip3 install packages/simcore-sdk[test]
-        pushd services/director; pip3 install -r requirements/ci.txt; popd
-        pip3 install packages/director-sdk/python
-        pushd services/web/server; pip3 install -r requirements/ci.txt; popd
-        pushd services/storage; pip3 install -r requirements/dev.txt; popd
-        pip3 install -r api/tests/requirements.txt
-        pip3 install -r services/apihub/tests/requirements.txt
+        bash ops/travis/helpers/ensure_python_pip
+        pip3 install pylint~=2.0
```

Comment:
Well, here we are not installing anything but just running pylint ... on all the *.py files
The idea is that this will be replaced by a test owned by each module. That is already included in the cookie-cutter for services (see e.g. [test_package.py](https://github.com/ITISFoundation/cookiecutter-osparc-service/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/tests/unit/test_package.py#L26))